lib/spack/spack/package.py

```diff
@@ -483,37 +483,67 @@ class SomePackage(Package):
     #
     # These are default values for instance variables.
     #
-    """By default we build in parallel.  Subclasses can override this."""
     parallel = True
+    """By default we build in parallel.  Subclasses can override this."""
```

Comment:
What about the multi-line ones below? I feel like a docstring makes more sense for a multi-line comment.